src/store/modules/TsModsModule.ts

```diff
@@ -192,6 +176,28 @@ export const TsModsModule = {
         },
         updateDeprecated(state, allMods: ThunderstoreMod[]) {
             state.deprecated = Deprecations.getDeprecatedPackageMap(allMods);
+        },
+        prewarmCacheMod(state: State, mods: ThunderstoreMod[]) {
+            const localState = new Map<string, CachedMod>(state.cache.entries());
+            mods.forEach(mod => {
+                const cacheKey = `${mod.getName()}-${mod.getVersionNumber()}`;
+
+                if (localState.get(cacheKey) === undefined) {
+                    const tsMod = state.mods.find((m) => m.getFullName() === mod.getName());
+
+                    // Updating Vuex state directly instead of mutations is a bad
```

Comment:
After the changes this comment is incorrect in at least three different ways and should be removed.